### DISCUSSION TITLE
RSE daily catchup

### DISCUSSION THREAD

**Discussion by matthewcoole** [NERC-CEH/rse_group#11] [General]
Just to get the ball rolling - what are people's thoughts on how to structure the daily RSE catchups?

Couple of options we've kind of already discussed:
- Treating the meeting like a stand-up (e.g. update from each, plan for the day, announcements).
  - This would keep the meeting short
  - May feel a bit impersonal with regards to team building
- Starting a separate updates file/board and keeping the meeting as an informal chat. 
  - Has the advantage of being asynchronous if people can't attend the meeting.
  - Creates a bit of a log for us (without anyone having to take meeting minutes).
  - Lack of structure may lead to meetings going on longer than they need.

Any other thoughts or ideas from you guys please discuss :)

As well as this should we create a regular schedule for our attendance to the EDS stand-up?

**Comment by metazool** [comment]
Thankyou for getting this started! I like the idea of discussions as a low-effort log for our future selves, and also as a basis for weeknotes. The more work in public we undertake, the easier it gets to maintain links across projects (thus the nudging to open the plankton repo)

In the shorter term, I don't find relative lack of structure a problem, for team building - if we find that the daily catchups end up freewheeling or burdensome we could bring topics to them in advance, sometimes deliberately schedule longer ones, taper the frequency over time.

If the purpose of the large group EDS meetings switches to more of a "scrum of scrums" (sorry) then having a rotation of visits to it would make sense to me.

**Comment by mattjbr123** [comment]
Yeah I agree with @metazool about the keeping it relatively informal for now for team-building and seeing how that goes for a bit, and having a schedule of who goes to the larger EDS group standups to feed back to the RSE team.

**Comment by jmarshrossney** [comment]
Hi all! 

I broadly agree that Github is a good place to have some or even most discussions in public, and really like the structure suggested in #16.

I've slightly different personal interests/preferences which would lead me to use the public space less often than suggested here in the absence of collective responsibility, but happy to adapt and see how things go.

My main concern is that many of our discussions will be related to specific issues and people connected to projects we are working on, and we need to be realistic about the risks of having these discussions in public. I don't know how much of our discussion will be of this nature but if it turns out to be significant we could keep the daily catch up (and, sigh, Teams) for these types of discussions.

An alternative would be to have a private repo with Discussions enabled which together with the daily/weekly chat form the core of our discussion. Then a separate public repo/discussion for anything outward-facing. I understand the dislike of having two separate discussion areas but to me it actually makes more sense to lower friction in the private discussion space, and have more curated discussions in the public space which are actually relevant to potentially external collaborators.

Happy to discuss more! :)

**Comment by metazool** [comment]
https://github.com/NERC-CEH/rse_group/discussions/12 is an example of an early-stage project discussion (intro to the research and identifying how we could offer support in both short and long term) that _might_ involve sensitivity about the field site. In this case I asked the researcher first about taking a public note

If we are thinking about a forum like this for supporting the governance process for projects obtaining support from our group then I could see some sensitivity around that - people being reluctant to have a public trail of their proposals not making the cut for whatever reason. If the aim is to facilitate a peer review process for proposals, falling back to Teams might be preferable as permissions management is taken care of. It will be a nice problem to have.

**Comment by metazool** [comment]
While the text summaries are helpful for those of us (ok, me) who find it easier to digest written than spoken information, their present form lends a "reporting circle" vibe to the occasion.

The standups I've most enjoyed being in have gone round topics, not people, with the team taking turns to faciliate the walkthrough. A close analogue is the "Walk the Board" / "Rotate the Facilitator" pattern described here - https://martinfowler.com/articles/itsNotJustStandingUp.html#walktheboard 

It does need one place to look, though! I thought that was a part of the intention in trialling this Discussions forum, that we'd leave notes on topics in hand and the active ones would float to the top.

**Comment by mattjbr123** [comment]
That's a nice guide, thanks @metazool 
I like the idea of topics instead of people, to avoid the 'I did that, I'm doing this, I will do something else...' repetition. But yeah, it would require us essentially posting in this discussion repo to get that 'one place to look' thing, which I think we might need initially so that the pressure is not on an individual to come up with a discussion topic, but it's just there 'on the board'. 
I'd be less keen on turning it into a sports-like team huddle though xD

**Comment by longr** [comment]
The product, project, meeting and legal obligations rolling minute template used by the [eScience Lab](https://esciencelab.org.uk/) at the University of Manchester for it's weekly team meeting.
https://zenodo.org/records/12583986
